docs/conf.py

```diff
@@ -95,7 +95,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
```

Comment:
Ah we are not using that at the moment but I think it's ok to leave it as is in case we want to incorporate some pictures in the future. Thanks for pointing that out

Reply:
The default behavior is to set it as _static if it's not already set and removing it will get rid of the warning that I mentioned.

Reply:
ah ok, good to know. Thank you. I will merge it now.